mobile-app/cypress/integration/functional/wallet/dex/remove_liquidity.spec.ts

```diff
@@ -12,9 +12,9 @@ function createAddLiquidityToWallet (): void {
 
   cy.getByTestID('liquidity_screen_list')
     .wait(2000).getByTestID('pool_pair_row_your').first()
-    .invoke('text').should(text => expect(text).to.contains('10.00000000 dETH-DFI'))
+    .invoke('text').should(text => expect(text).to.contains('dETH – DFI10.00000000'))
```

Comment:
is this expected?

Reply:
Ya because previously the UI uses token string as suffix, but the UI replaces it with an icon.
I can change to just check the number string if this looks confusing